src/components/eip4626/user/VaultDeposit.tsx

```diff
@@ -29,6 +29,7 @@ export function VaultDeposit({ vaultAddress }: VaultInfoProps) {
   const { data: vaultAssetUserBalance, error: vaultAssetUserBalanceError } =
     useReadBalanceOf(vaultAssetAddress as EvmAddress);
 
+  //@TODO dynamic decimals
```

Comment:
Why can't we do in scope of this PR?

Reply:
Don't want to expand the scope of the PR anymore, it's big enough.